samples/highcharts/series-dumbbell/lowmarker/demo.js

```diff
@@ -0,0 +1,204 @@
+// Define custom SVG symbols for left and right triangles
+Highcharts.SVGRenderer.prototype.symbols.triangleLeft = function (x, y, w, h) {
+    return ['M', x, y + h / 2, 'L', x + w, y, 'L', x + w, y + h, 'Z'];
+};
+
+Highcharts.SVGRenderer.prototype.symbols.triangleRight = function (x, y, w, h) {
+    return ['M', x + w, y + h / 2, 'L', x, y, 'L', x, y + h, 'Z'];
+};
+
+// Process and transform the data for the chart
+const data = [{
+    name: 'wall-breakers',
+    previous: 47,
+    current: 128
+}, {
+    name: 'electro-giant',
+    previous: 99,
+    current: 88
+}, {
+    name: 'mortar',
+    previous: 66,
+    current: 79
+}, {
+    name: 'hog-rider',
+    previous: 84,
+    current: 78
+}, {
+    name: 'royal-giant',
+    previous: 104,
+    current: 73
+}, {
+    name: 'graveyars',
+    previous: 101,
+    current: 70
+}, {
+    name: 'test',
+    previous: 51,
+    current: 50
+}, {
+    name: 'giant',
+    previous: 41,
+    current: 49
+}, {
+    name: 'royal-hogs',
+    previous: 39,
+    current: 48
+}, {
+    name: 'balloon',
+    previous: 62,
+    current: 47
+}, {
+    name: 'lava-hound',
+    previous: 54,
+    current: 42
+}, {
+    name: 'goblin-drill',
+    previous: 19,
+    current: 41
+}, {
+    name: 'golem',
+    previous: 49,
+    current: 40
+}, {
+    name: 'x-bow',
+    previous: 55,
+    current: 36
+}, {
+    name: 'miner',
+    previous: 9,
+    current: 31
+}, {
+    name: 'battle-ram',
+    previous: 48,
+    current: 30
+}, {
+    name: 'goblin-giant',
+    previous: 10,
+    current: 30
+}, {
+    name: 'three-musketeers',
+    previous: 18,
+    current: 28
+}, {
+    name: 'goblin-barrel',
+    previous: 57,
+    current: 21
+}, {
+    name: 'ram-rider',
+    previous: 23,
+    current: 20
+}, {
+    name: 'elixir-golem',
+    previous: 4,
+    current: 7
+}, {
+    name: 'skeleton-barrel',
+    previous: 4,
+    current: 3
+}].map(dataPoint => {
+    const isIncrease = dataPoint.previous < dataPoint.current;
+    return {
+        ...dataPoint,
+        low: isIncrease ? dataPoint.previous : dataPoint.current,
+        high: isIncrease ? dataPoint.current : dataPoint.previous,
+        color: isIncrease ?
+            Highcharts.getOptions().colors[2] :
+            Highcharts.getOptions().colors[5]
```

Comment:
Since all the increasing points are assigned to the same series, and all the decreasing ones to another, I assume it would be better to set the color on the series rather than on each individual point.